api/system-settings/getOtpToken

```diff
@@ -0,0 +1,70 @@
+#!/usr/bin/perl
+
+#
+# Copyright (C) 2020 Nethesis S.r.l.
+# http://www.nethesis.it - nethserver@nethesis.it
+#
+# This script is part of NethServer.
+#
+# NethServer is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License,
+# or any later version.
+#
+# NethServer is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with NethServer.  If not, see COPYING.
+#
+
+use warnings;
+use strict;
+use JSON;
+use esmith::ConfigDB;
+use NethServer::ApiTools;
+use Convert::Base32;
+
+my $db = esmith::ConfigDB->open_ro;
+my $input = NethServer::ApiTools::readInput();
+my $username = $input->{'username'} || '';
+my $OtpStatus;
+my $key;
+my $domainName = $db->get_value('DomainName');
+my $systemName = $db->get_value('SystemName');
+
+my $file = '/var/lib/nethserver/2fa/'.$username;
+if (! $file || ! -f $file ) {
```

Comment:
The `file` variable can't be empty, you just initialized it ;)
```suggestion
if (! -f $file ) {
```

Reply:
done